pkg/controller/storagecluster/reconcile.go

```diff
@@ -1237,7 +1251,11 @@ func newCleanupJob(sc *ocsv1.StorageCluster) *batchv1.Job {
 									ReadOnly:  true,
 								},
 							},
-							Command: []string{"bash", "-c", "ceph osd out osd.${FAILED_OSD_ID};ceph osd purge osd.${FAILED_OSD_ID} --force --yes-i-really-mean-it"},
+							Command: []string{
+								"/bin/bash",
+								"-c",
+								fmt.Sprintf(purgeOSDScript),
```

Comment:
no need to format it
```suggestion
								purgeOSDScript,
```

Reply:
okay. I will change it